code/go/0chain.net/miner/chain.go

```diff
@@ -248,6 +250,52 @@ func (mc *Chain) SetLatestFinalizedBlock(ctx context.Context, b *block.Block) {
 	}
 }
 
+// LoadLatestBlocksFromStore loads LFB and LFMB from store and sets them
+// to corresponding fields of the sharder's Chain.
+func (mc *Chain) LoadLatestBlocksFromStore(ctx context.Context) error {
+	// var bl *blocksLoaded
+	lfbr, err := mc.LoadLFBRound()
+	if err != nil {
+		return fmt.Errorf("load_lfb - could not load lfb from state DB, err: %v", err)
+	}
+
+	logging.Logger.Debug("load_lfb - load from stateDB",
+		zap.Int64("round", lfbr.Round),
+		zap.String("block", lfbr.Hash))
+
+	// fetch from sharders
+	b, err := mc.GetNotarizedBlockFromSharders(ctx, lfbr.Hash, lfbr.Round)
```

Comment:
do we always want to load this block? it can be stored locally

Reply:
Miners do not store blocks, and this `LoadLatestBlocksFromStore` will only be called once when start the miner, so should be fine.